src/index.md

```diff
@@ -1370,6 +1370,139 @@ The fix for the main issue should remove the test from the ProblemList or remove
+   * Clone **(O)** and add the prefix `[REDO]` on the summary - the clone becomes the redo-issue **(R)**.
+   * Make sure relevant information is brought to **(R)**.
+   * Remember that comments aren't brought over when cloning.
+#. Create a backout-issue **(B)**:
+   * Alternative 1 - the regression is identified directly.
+     * Create a sub-task to **(R)** with the same summary, prefix with `[BACKOUT]`.
+   * Alternative 2 - an investigation issue is created **(I)**, and during the investigation backing out the change is identified as the best solution.
+     * Use the investigation issue **(I)** for the backout.
+     * Change summary of **(I)** to the same as **(O)** and prefix with `[BACKOUT]`.
+     * Move and change type of **(I)** to become a sub-task of **(R)**.
+   * Alternative 3 - no redo issue was created.
+     * Create a backout-issue **(B)** with the same summary as **(O)**, prefix with `[BACKOUT]`.
+     * Link **(B)** and **(O)**.
+
+ProblemList entries and `@ignore` keywords will continue to point to the original bug (unless updated at back out). This is accepted since there is a clone link to follow.
+
+### How to work with git when a change is backed out
+
+To backout a change with git, use `git revert`. This will apply (commit) the anti-delta of the change.
+
+~~~diff
+$ git show aa371b4f02c2f809eb9cd3e52aa12b639bed1ef5
+commit aa371b4f02c2f809eb9cd3e52aa12b639bed1ef5 (HEAD -> master)
+Author: Jesper Wilhelmsson <jesper.wilhelmsson@oracle.com>
+Date:   Wed Jun 23 20:31:32 2021 +0200
+
+    My change
+
+diff --git a/README.md b/README.md
+index 399e7cc311f..4961acb2126 100644
+--- a/README.md
++++ b/README.md
+@@ -1,4 +1,4 @@
+-# Welcome to the JDK!
++# Welcome to my modified JDK!
+
+ For build instructions please see the
+ [online documentation](https://openjdk.java.net/groups/build/doc/building.html),
+
+$ git revert aa371b4f02c2f809eb9cd3e52aa12b639bed1ef5
+[master d454489052d] Revert "My change"
+ 1 file changed, 1 insertion(+), 1 deletion(-)
+
+$ git show d454489052dc6ff69a21ad9c8f56b67fdeb435ee
+commit d454489052dc6ff69a21ad9c8f56b67fdeb435ee (HEAD -> master)
+Author: Jesper Wilhelmsson <jesper.wilhelmsson@oracle.com>
+Date:   Wed Jun 23 20:32:08 2021 +0200
+
+    Revert "My change"
+
+    This reverts commit aa371b4f02c2f809eb9cd3e52aa12b639bed1ef5.
+
+diff --git a/README.md b/README.md
+index 4961acb2126..399e7cc311f 100644
+--- a/README.md
++++ b/README.md
+@@ -1,4 +1,4 @@
+-# Welcome to my modified JDK!
++# Welcome to the JDK!
+
+ For build instructions please see the
+ [online documentation](https://openjdk.java.net/groups/build/doc/building.html),
+~~~
+
+### How to work with mercurial when a change is backed out
+
+In order to backout a change, the `hg backout` command is recommended, which essentially applies the anti-delta of the change. Make sure you perform the backout in the most upstream repostiory the change has escaped to.
```

Comment:
typo: repostiory

Reply:
Thank you for reviewing! Fixed the typo.